routes/send.js

```diff
@@ -45,6 +45,12 @@ exports.send = {
       send(server, mailObj, details, done) {
         server.methods.sendMail(mailObj, details.sendIndividual, done);
       },
+      track(server, details, send, done) {
+        if (server.settings.app.ENV.MICRO_METRICS_HOST) {
```

Comment:
can we create another key for `metricsEnabled: {{ENV.MICRO_METRICS_HOST ? true: false}}` and use that everywhere?